src/main.ts

```diff
@@ -1,5 +1,7 @@
 import {app, BrowserWindow} from "electron";
 
+const EXCALIDRAW_BUNDLE = `file://${__dirname}/../excalidraw-2020-02-29-10-52-11.asar/index.html`;
```

Comment:
I'm kind of hesitate to include this bundle in the repo though.. It will be changing quite often

Reply:
What would be the alternative for the Initial Version? Surely it would then update it Always with the newest Version as described in #5

Reply:
Well the other alternative is to use the online version for the first login and download it in the background.

Reply:
Could you download the latest version when building the app?

Reply:
I thought of that as well..! Maybe that's the best one :) We can also get rid of the version from the asar file and it will always be something like `https://excalidraw.com/excalidraw.asar`..

We can have a script that does it for us and puts it wherever we want..

Reply:
https://github.com/excalidraw/excalidraw/pull/857

Reply:
That would work I guess :+1: